src/components/Independent/Independent.js

```diff
@@ -58,18 +52,24 @@ const Independent = ({ tiktoksrc, addPin, removePin, removeTikTok }) => {
     >
       <Card
         elevation={6}
-        id={ tiktoksrc.substring(23, 63) }
         >
-        {error && <CardContent className="error"> {error}, Cannot retrieve from TikTok!</CardContent>}
-        <CardMedia 
-          image={ thumbnail }
-          component="img"
-          aria-label={ `Thumbnail of ${ title }` }
-        />
-        <CardContent>
-          <a href={ authorLink } className="author">@{ author }</a>
-          <p className="title">{ title }</p>
-        </CardContent>
+        {status_msg 
+        ? <CardContent>
+            <p>Not able to load this tiktok, please check your path url</p>
+            <p className="error-address">{ cite }</p>
+          </CardContent>
+        : <>
+            <CardMedia 
+              image={ thumbnail_url }
+              component="img"
+              aria-label={ `Thumbnail of ${ title }` }
+            />
+            <CardContent>
+              {author_name && <p className="author">@{ author_name }</p>}
+              {title && <p className="title">{ title }</p>}
+            </CardContent>
+          </>
+        }
```

Comment:
Nice. I do like ternaries, but I wonder if this is big enough that it needs to be pulled out into a method.

Reply:
The initial thought was to conditionally render the error content as usual but I thought the styling that came with using the CardContent component flowed better with the rest of the application

But to your point breaking this out would at least knock out this ternary into an if conditional and default return instead of being adamant about the ternary

Reply:
I try to only use ternaries if they're fairly simple. I don't subscribe to Leta's philosophy of only one line, but I try not to get too wild with it.